xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -203,48 +197,47 @@ bool CGUIDialogPVRGuideInfo::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
-void CGUIDialogPVRGuideInfo::SetProgInfo(const CFileItem *item)
+void CGUIDialogPVRGuideInfo::SetProgInfo(const EPG::CEpgInfoTagPtr &tag)
 {
-  *m_progItem = *item;
+  m_progItem = tag;
```

Comment:
This changes behavior. Instead of the dialog having it's own copy, it now holds a pointer to some mutable global value. So, are you sure it isn't changed/passed to something?